packages/ERTP/src/markObjects.js

```diff
@@ -0,0 +1,35 @@
+import { Far } from '@agoric/marshal';
+
+/** @type {ERTPKind} */
+export const ERTPKind = {
+  ISSUER: 'issuer',
+  BRAND: 'brand',
+  PURSE: 'purse',
+  PAYMENT: 'payment',
+  MINT: 'mint',
+  // TODO: improve implementation such that spaces can be used in
+  // ERTPKind. Currently we assume that the ERTPKind is the last
+  // element of the interface, after we split on spaces.
+  DEPOSIT_FACET: 'depositFacet',
+};
+
+/** @type {MakeFarName} */
+export const makeFarName = (allegedName, kind) => `${allegedName} ${kind}`;
+
+export const markBrand = (allegedName, brand) =>
+  Far(makeFarName(allegedName, ERTPKind.BRAND), brand);
```

Comment:
As an example

```suggestion
export const markBrand = (allegedName, brand) =>
  Far(`${allegedName} brand`, brand);
```

This simplified function is also self contained and used in only one place in brand.js. Likewise each of the functions below are called in only one place in the file that is about the making of that thing. Is any purpose served by gathering this functions into this file, rather than moving each into the file where it is used? 

IIUC, that migration would be more in the spirit of what you're trying to accomplish with this PR --- organizing the code according to the kind of object that code is about. If a seventh Far type is added to ERTP, if it is sufficiently separate, we'd hope that most of the logic of that type is in its file, and the other files that need to be modified are those that need to know about that type. OTOH we centralize package-wide knowledge in our types.js, exports.js, index.js, so maybe we should consider this a similar centralization of package-wide knowledge.

(Since I wrote this @katelynsills and I discussed verbally and came to agreement.)

Reply:
Just took this out entirely.